src/GeneExpressionTSnePlot.js

```diff
@@ -173,7 +176,7 @@ const GeneExpressionScatterPlot = (props) => {
   return [
     <AtlasAutocomplete
       key={`expression-autocomplete`}
-      wrapperClassName={`row`}
+      wrapperClassName={`row gene-search-container`}
```

Comment:
I’m thinking that besides a class name prop components should expose a styles attribute, too. Otherwise you need to rely on an external CSS to also add rules for `gene-search-container`. One option we have here is to use Styled Components and add the class `gene-search-container` with `createGlobalStyle`. It’s already part of our dependencies, so the bundles would be unaffected by the change.